pkg/pgmodel/pgx.go

```diff
@@ -677,19 +690,64 @@ func (q *pgxQuerier) HealthCheck() error {
 	return nil
 }
 
+func (q *pgxQuerier) Select(mint int64, maxt int64, sortSeries bool, hints *storage.SelectHints, ms ...*labels.Matcher) (storage.SeriesSet, storage.Warnings, error) {
```

Comment:
Sinve all of this stuff seems to be non-pgx specific, should it go in `querier.go`, or another file, instead?